cmd/replay/main.go

```diff
@@ -280,6 +280,7 @@ func main() {
 		}
 
 		validatingAgainstEspresso := arbos.IsEspressoMsg(message.Message) && chainConfig.ArbitrumChainParams.EnableEspresso
+		validatingAgainstCentrailized := arbos.IsL2NonEspressoMsg(message.Message) && chainConfig.ArbitrumChainParams.EnableEspresso
```

Comment:
I think we don't really care if the alternative sequencer is centralized so I suggest not using that term. Not sure what's best but how about something like `validatingEspressoLivenessFailure`?